Dockerfile

```
@@ -0,0 +1,14 @@
+# ----- EIB Builder Image -----
+FROM golang:1.21
```

you should use the bci-golang here to stay consistent:
```suggestion
FROM registry.suse.com/bci/golang:1.21
```

---

I didn't realize this was a thing, but I dig it. Thanks for the heads up :)